### DISCUSSION TITLE
RLC-520A: IPC_523128M5MP_V2 v3.1.0.2368 build 23062701

### DISCUSSION THREAD

**Discussion by Dnominated** [AT0myks/reolink-fw-archive#33] [Firmware sharing]
Latest in the archive shows build 22082201 for RLC-520A hw 523128M5MP v2.
Build 22100622:
https://support-d.reolink.com/attachments/token/72ZbJjwx7YB3xvZP1fYl1cD7t/?name=IPC_523128M5MP_V2.1387_22100622.RLC-520A.OV05A10.5MP.REOLINK.pak
From: https://community.reolink.com/topic/3792/rlc-520a-v2-firmware/23?post_id=22939&lang=en-US
Tested on the hw listed above. Seems to work fine.

Build 23062701:
https://support-d.reolink.com/attachments/token/b32M4tyle4rDpYdNSATF25tZP?name=IPC_523128M5MP_V2.2368_23062701.RLC-520A.OV05A10.5MP.REOLINK.pak
Seems to work, **but the RLN8-410-E H3MB16 (oldest hw version) with fw 2.0.0.274_20120701 gives a 'connection error'** with this firmware. Re-flashed camera and reset, to build 22082201, then re-upgraded to this build. Same issue. NVR detects the camera on port 9000 - If the port is changed to 9001, it detects the camera operating on the new port, but maintains the 'Connection error' status. Everything else (Reolink app, web access) seems to work completely normally with this build. Worth noting that this is the oldest 8-channel NVR that Reolink made. I found the v2.0.0.280_21060101 build on AT0mykcs, and upgraded/flashed and reset the NVR, then attempted flashing the RLC-520A v2 to 23062701 again. **This time it worked.**

Note: it's probably not necessary to reset the NVR, to get the RLC-520Av2 Build 23062701 to work with it. I did it by accident, when choosing the 'Update Configurations' option, when updating the flash to v2.0.0.280_21060101. I didn't realize it would reset the NVR settings, like the lower choice 'Default Settings' and Restore button do.

**Comment by AT0myks** [comment]
Thanks, v3.1.0.1387_22100622 has been added in e9c8ceadb816dcf860abf88b47b079244f78cd19.
This v3.1.0.2368_23062701 link has already been shared in #35 and the first one in #25 but they've only been added very recently in 3ec625a6d1a6e413d5ef23f5977799bb3e6942cc and afd6ac08cc9a8de799dd07fa4ed315829036fb0d so that's why this version didn't show up at the time.

**Reply by lionleo** [reply]
New firmware
https://support-d.reolink.com/attachments/token/x08EPptoOMN3v8nNxk2MhVbDN/?name=IPC_523128M5MP_V2.2109_23082402.RLC-520A.OV05A10.5MP.REOLINK.pak

**Reply by wild-a** [reply]
> New firmware https://support-d.reolink.com/attachments/token/x08EPptoOMN3v8nNxk2MhVbDN/?name=IPC_523128M5MP_V2.2109_23082402.RLC-520A.OV05A10.5MP.REOLINK.pak

This is older than the one shared in #35. That one is 2368, the one you've shared is 2109.

**Comment by lukasMega** [comment]
how to download the firmware from URLs above? 
 - because the URLs redirecting to reolink.zendesk.com/…
<img width="611" alt="scr 2025-05-01 at 18 06 52" src="https://github.com/user-attachments/assets/9da0f6e5-c143-4cec-938a-6990f364ac3a" />
